src/Services/Analytics.ts

```diff
@@ -0,0 +1,96 @@
+import { GA } from "../Utils/GA";
+
+type Options = {
+  ga: {
+    trackingId: string;
+    appName: string;
+    env: string;
+  };
+};
+
+type TransferEventData = {
+  address: string;
+  recipient: string;
+  amount: number;
+  nonce?: number;
+};
+
+class AnalyticsService {
+  ga: GA;
+
+  constructor(options: Options) {
+    this.ga = new GA(options.ga);
+  }
+
+  transferIntransitEvent({
```

Comment:
Each method should have a verb. Please, rename to `trackTransferIn**T**ransit`.

Apply the same for all methods.

Reply:
fixed